en.json

```diff
@@ -250,15 +250,15 @@
         "Options.SeatedMode.Off" : "Seated Mode: Off",
 
         "Indicator.OnlineUsers" : "Online Users: ~{num_total} ({num_registered})",
-        "Indicator.ServerStatus" : "<color=#fff>Server Status:</color> {status, select, Good {Good} Slow {Slow} Down {Down} NoInternet {No Internet}} ({response_ms} ms)",
+        "Indicator.ServerStatus" : "<color=#fff>Server Status:</color> {status, select, Good {Good} Slow {Slow} Down {Down} No Internet {No Internet}} ({response_ms} ms)",
```

Comment:
This needs to be "NoInternet" it's a keyword match, with the actual displayed message being in the braces { }

Reply:
My mistake, I thought it was the other way around (with the braces being the actual "variable/keyword").